components/support/utils/src/main/java/mozilla/components/support/utils/URLStringUtils.kt

```diff
@@ -101,4 +101,33 @@ object URLStringUtils {
     // complain that this value is not one of the predefined enums that are allowed.
     @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
     internal var flags = 0
+
+    private const val HTTP = "http://"
+    private const val HTTPS = "https://"
+    private const val WWW = "www."
+
+    /**
+     * This method uses the un-shortened URL to generated a shortened URL
```

Comment:
nit: to generate, not generated.

Maybe let's rephrase this to:

Generates a shorter version of the provided URL for display purpose by stripping it of...